source/helpers/Objects.tsx

```diff
@@ -44,3 +44,10 @@ export async function deepCompareEquals(
 
   return firstHash === secondHash;
 }
+
+/**
+ * Checks if a variable is an object
+ * @param i any variable
+ */
+export const isObject = (i: any): i is Record<any, any> =>
+  !!i && i.constructor === Object;
```

Comment:
I wonder if the typeguard is needed. Using `Record<any, any>` doesn't really helps us either. How about making a typeguard that explicitly tells us it is an answer object instead?

Reply:
As the only definition of an answer object is as `Record<string, Answer>` parameters to functions, and the `Answer` type doesn't match the structure that we have where this typeguard is used, this wouldn't make sense. 

https://github.com/helsingborg-stad/app-mitt-helsingborg/blob/16abbde25b39c9d83d38a0a2188d27b41ad3707d/source/types/CaseContext.ts#L36

https://github.com/helsingborg-stad/app-mitt-helsingborg/blob/16abbde25b39c9d83d38a0a2188d27b41ad3707d/source/types/CaseContext.ts#L23-L26

Checking for a `Record<any, any>`, is fine, as an Object can have any type as its keys and values.

Reply:
Well, instead of `Record<any, any>` the more correct type would be `Record<string, unknown>` since we don't use `any`.

Reply:
In that case, wouldn't `Record<unknown, unknown>` be better suited? Again, keys don't have to be string values for the object to be valid.